content/en/01-about-pixie/04-architecture.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Architecture"
+metaTitle: "About Pixie | Architecture"
+metaDescription: "What Pixie deploys to your cluster and what each component does."
+order: 4 
+---
+
+## Main Components 
+
+Pixie's deploy consists of three main components:
+1. Vizier
+    - Vizier is the core of Pixie. It is responsible for collecting and processing data in a cluster. 
+2. Pixie Operator
+    - The Pixie operator is responsible for managing a Vizier instance. It deploys Vizier, tracks its state, keeps it up-to-date, and helps auto-recover in bad situations.
+    - The latest version of the Pixie operator is automatically deployed by OLM once OLM is up and running.
+3. [Operator Lifecycle Manager](https://olm.operatorframework.io/)
+    - OLM is responsible for deploying the Pixie operator, and keeping it up-to-date.
+    - OLM comes with a `olm` namespace by default. The resources in here are responsible for keeping OLM running.
+    - Pixie additionally deploys the `px-operator` namespace. Most of the jobs/pods in here are deployed by OLM and are responsible for checking for updates to the Pixie operator and re/deploying the Pixie operator.
```

Comment:
I think if we are going to list out namespaces here, we may want to go all in and have a section that lists the k8s resources that are created. maybe not every single thing but like what goes in X namespace, service, etc. 

To me, this reads as TMI on the OLM section and not enough info on the Vizier section.

I know the more info comes later but we may want to reorder it.

Reply:
Perhaps something like,

Logical components: Vizier & Cloud
What does each thing responsible for?

Deployment: Talk about the operator, then go into the specifics of what is deployed